src/util/backend.ts

```diff
@@ -0,0 +1,57 @@
+import { BACKEND_URL } from './constants';
+
+export interface IErrorResponse {
+	code: number;
+	message: string;
+}
+
+async function makeBackendRequest(
+	endpoint: string,
+	method: 'get' | 'post',
+	jwt: string | null,
```

Comment:
Handle this internally in the function or create a wrapper which handles authenticated requests

Reply:
Will be handled in https://github.com/kossiitkgp/KWoC-Frontend/issues/149.